### DISCUSSION TITLE
How many 🍎🍏 were originally in the box?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#858] [Galaxy Trivia] [answered]
On a recent expedition, three intrepid adventurers found themselves stranded in the middle of the desert with only a box of apples. During the night, @lucassanttoss woke up, decided to hide some of the apples, hid a third, and promptly fell asleep again. @priyanshugour70 woke up a short time later, decided to hide a third of the remaining apples, and fell asleep again. Finally, @husamettinarabaci woke up, saw the others asleep, and took a third of the remaining apples. Of course, none of the adventurers knew of the other's antics, so in the morning they divided the remaining apples, each getting 16.

**Comment by husamettinarabaci** [comment]
Wow. I guess I'm the unluckiest thief :D Please give back my remaining 14 apples. @lucassanttos  @priyanshugour70

**Comment by peterbaccaro** [comment, marked as answer]
There were originally **162 apples** in the box.

Working backwards:
- At the end there were 16 apples each, therefore there were 48 apples in total.
- Charlie removed one third leaving 48, so there must have been 72 apples before.
- Blake removed one third leaving 72, so there must have been 108 apples before.
- Alex removed one third leaving 108, so there must have been 162 apples before.

So:
- Alex hid 54, leaving 108.
- Blake then hid 36, leaving 72.
- Charlie then hid 24, leaving 48 available to share in the morning.

**Comment by ParanoidUser** [comment]
Round of applause for all participants! 👏👏
